docs/moonlight/modifiers.md

```
@@ -22,6 +22,6 @@ The following Modifiers are defined in MoonLight. Some found there origin in WLE
 | Rotate | ![Rotate](https://github.com/user-attachments/assets/c622a9df-318a-4f83-81c0-f5a5c7bafb7b) | <img width="320" alt="Rotate" src="https://github.com/user-attachments/assets/7273a625-5e14-4814-92e8-8f7f65cfc02e" /> | |
 | Checkerboard | ![Checkerboard](https://github.com/user-attachments/assets/54970267-35af-406c-9558-c1f4219a71c0) | <img width="320" alt="Checkerboard" src="https://github.com/user-attachments/assets/66d51dc7-b816-4ca7-b1e3-57b067566516" /> | |
 | Pinwheel 🧊 | ![PinWheel](https://github.com/user-attachments/assets/e5dbadbe-eeb1-41e5-b197-ec4bd5366aea) | <img width="320" alt="PinWheel" src="https://github.com/user-attachments/assets/46585cea-d301-4221-9af2-65f8054543da" /> | Projects 1D/2D effects onto 2D/3D layouts in a pinwheel pattern.<br>**Swirl**: bend the pinwheel<br>**Rotation Symmetry**: rotational symmetry of the pattern<br>**Petals** Virtual width<br>**Ztwist** twist the pattern along the z-axis<br>Height: distance from center to corner |
-| RippleYZ 🧊 | ![RippleYZ](https://github.com/user-attachments/assets/0918efac-6367-420f-b0e3-d796d9551953) | <img width="320" alt="RippleYZ" src="https://github.com/user-attachments/assets/90ecf22c-c4c1-4ee9-8096-fd5613fbb1a7" /> | 1D/2D effect will be rippled to 2D/3D (🚨)<br>Shrink: shrinks the original size towards Y and Z, towardsY: copies X into Y, towardsZ: copies XY into Z |
+| RippleXZ 🧊 | ![RippleXZ](https://github.com/user-attachments/assets/0918efac-6367-420f-b0e3-d796d9551953) | <img width="320" alt="RippleXZ" src="https://github.com/user-attachments/assets/90ecf22c-c4c1-4ee9-8096-fd5613fbb1a7" /> | 1D/2D effect will be rippled to 2D/3D (🚨)<br>Shrink: shrinks the original size towards X and Z, towardsX: copies Y into X, towardsZ: copies XY into Z |
```

markdownlint-cli2 (0.18.1)

Table pipe style
Expected: leading_only; Actual: leading_and_trailing; Unexpected trailing pipe

(MD055, table-pipe-style)